tools/sggolangcilint/golangci.yml

```diff
@@ -265,6 +265,7 @@ linters-settings:
 
   goconst:
     ignore-tests: true
+    ignore-strings: "```"
```

Comment:
Since that string is used multiple times in some files (typically `"```" + someStringVar + "```"`), it triggers the goconst linter.

We need to either:
- declare `"```"` as a variable / constant when triggering
- wave the linter check at every offending line
- wave `"```"` everywhere for goconst (which is done here)

Reply:
could you link to an example? seems like an odd case, we don't want to add too many overrides here - different repos have different such "uniquenesses"

Reply:
I believe it was [here](https://github.com/einride/sage/blob/73a71b889ba1b67d2f3c3333b688ffe892391922/tools/sgterraform/tools.go#L53).

Maybe it's okay to waive the check there.

Reply:
Actually a search found only 1 occurence of such case. I was too quick on the trigger.
I'll waive it at the offending line.